src/styles.scss

```diff
@@ -65,6 +65,13 @@ body {
     max-width: 80%;
   }
 
+  .task-name {
```

Comment:
Why do we need this in the global `style.scss`? 
If it is only for one html, put that in the scss under the same folder